.reuse/dep5

```diff
@@ -28,31 +28,35 @@ Files: ci/*
 Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
 License: Apache-2.0
 
-Files: docs/*
-Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+Files: clients/fossid-webapp/src/test/assets/*
```

Comment:
As the order does change the semantics, do you know if there is a way to compare the assigned licenses before and after your change using the reuse-tool?

Reply:
> As the order does change the semantics

Very good point, thanks for paying attention here!

> do you know if there is a way to compare the assigned licenses before and after your change using the reuse-tool?

I just simply ran `reuse spdx` before and afterwards, and compared the output. And I indeed discovered and fixed an issue (also see the updated commit message).